src/Security/Authorization/Core/src/AuthorizationOptions.cs

```diff
@@ -24,21 +24,26 @@ public class AuthorizationOptions
     public bool InvokeHandlersAfterFailure { get; set; } = true;
 
     /// <summary>
-    /// Gets or sets the default authorization policy. Defaults to require authenticated users.
+    /// Gets or sets the default authorization policy. Defaults to requiring authenticated users.
     /// </summary>
     /// <remarks>
-    /// The default policy used when evaluating <see cref="IAuthorizeData"/> with no policy name specified.
+    /// - The `DefaultPolicy` applies whenever authorization is required, but no specific policy is set.
+    /// - If an `[Authorize]` attribute is present without a policy name, the `DefaultPolicy` is used instead of the `FallbackPolicy`.
+    /// - This behavior ensures that endpoints explicitly requesting authorization (via `[Authorize]` or `RequireAuthorization()`) default to a secure policy.
+    /// - If a different default behavior is needed, developers should explicitly define policies rather than relying on `FallbackPolicy`.
```

Comment:
```suggestion
    /// - When non-default behavior is needed, developers should define named policies.
```

I don't think it's worth it to call out the `FallbackPolicy` again here. I think that's well covered by the second bullet point.